includes/template-library/sources/local.php

```diff
@@ -1292,10 +1292,25 @@ public function maybe_render_blank_state( $which ) {
 
 			$current_type = 'template';
 
-			$inline_style .= '#elementor-template-library-tabs-wrapper {display: none;}';
+			$args['additional_inline_style'] = '#elementor-template-library-tabs-wrapper {display: none;}';
 		}
 
+		$this->render_blank_state( $current_type, $args );
+	}
+
+	public function render_blank_state( $current_type, $args = [] ) {
 		$current_type_label = $this->get_template_label_by_type( $current_type );
+		$inline_style = '#posts-filter .wp-list-table, #posts-filter .tablenav.top, .tablenav.bottom .actions, .wrap .subsubsub { display:none;}';
+
+		$args = wp_parse_args( $args, [
+			'additional_inline_style' => '',
+			'href' => '',
```

Comment:
Where is the default `href`?

Reply:
The old code:
`esc_url( Utils::get_pro_link( 'https://elementor.com/pro/?utm_source=wp-custom-fonts&utm_campaign=gopro&utm_medium=wp-dash' ) ); `

was never really used since someone forget to use `echo` before it.

Reply:
OK